include/oneapi/tbb/flow_graph.h

```diff
@@ -385,23 +385,33 @@ class continue_receiver : public receiver< continue_msg > {
     template< typename R, typename B > friend class run_and_put_task;
     template<typename X, typename Y> friend class broadcast_cache;
     template<typename X, typename Y> friend class round_robin_cache;
+
+private:
     // execute body is supposed to be too small to create a task for.
-    graph_task* try_put_task( const input_type & ) override {
+    graph_task* try_put_task_impl( const input_type& __TBB_FLOW_GRAPH_METAINFO_ARG(const message_metainfo& metainfo) ) {
         {
             spin_mutex::scoped_lock l(my_mutex);
             if ( ++my_current_count < my_predecessor_count )
                 return SUCCESSFULLY_ENQUEUED;
```

Comment:
It seems like the meta is missed for this branch. As we discussed, if the `continue_node` did not receive a signal from each successor, it should store `metainfo` internally for the future use. In particular, we need to merge the metainfo with the previously stored one and reset it to the empty state once `execute` is called.

Reply:
Added metainfo merging. Test is coming up.